### PR TITLE
added unflattenObject

### PR DESCRIPTION
**Pull request by OlegPaska**

### CODE REVIEW

**Comment by OlegPaska** [comment]
image
flatten randomly becomes flattern in src\data-pipe.ts. I was going to change it but I didn't want to break anything. Flatten is the correct spelling

File: src/array/transform.ts [hunk 1]
```diff
@@ -53,6 +53,47 @@ export function distinct(array: any[], elementSelector?: Selector): any[] {
...
  * @returns 
  */
+
+export function unflattenObject(data: any): any {
+
+
+  function setProperty(data: any, pName: string, value: any): any {
+    const pNames = pName.split('.')
+    let parent: any = data
+    /**
```
Comment (ppaska) [inline src/array/transform.ts:63]: Remove unused code

File: src/array/transform.ts
Comment (ppaska) [inline src/array/transform.ts:1]: Add unflattenObject (and flattenObject) documentation to the readme.md

**Comment by OlegPaska** [comment]
fixed

**Pull request merged**